Documentation/configuration/sbom.rst

```diff
@@ -31,22 +31,17 @@ Prerequisites
 Download SBOM
 =============
 
-The SBOM can be downloaded from the supplied Cilium image using the
-``cosign download sbom`` command.
+You can download SBOM from the supplied Cilium image using
 
 .. code-block:: shell-session
 
-    $ cosign download sbom --output-file sbom.spdx <Image URL>
+    $ cosign download attestation --predicate-type spdxjson --output-file cilium_SBOM.spdx.json <Image URI>
 
-Verify SBOM Image Signature
-===========================
+Verify SBOM attestation
+=======================
 
-To ensure the SBOM is tamper-proof, its signature can be verified using the
-``cosign verify`` command.
+To verify SBOM attestation on the supplied Cilium image, run
```

Comment:
```suggestion
To verify the SBOM attestation for the supplied Cilium image, run the following command:
```

Also, it would be nice to add a sentence on what the SBOM “attestation” provides, as not all readers (or reviewers :innocent:) will be familiar with the concept. I suppose it's a signature to prove the authenticity and the integrity of the SBOM, right?

Reply:
> Also, it would be nice to add a sentence on what the SBOM “attestation” provides

I agree.

> To verify the SBOM attestation *for* the supplied Cilium image, run the following command:

I am following the command doc [here](https://github.com/sigstore/cosign/blob/main/doc/cosign_verify-attestation.md). I think there can be multiple attestations on an image.